jsonschema-core.md

```diff
@@ -1844,8 +1844,6 @@ this keyword applied a subschema. The value MAY be a boolean true if a subschema
 was applied to every index of the instance, such as is produced by the `items`
 keyword. This annotation affects the behavior of `items` and `unevaluatedItems`.
 
-Omitting this keyword has the same assertion behavior as an empty array.
```

Comment:
We still need to specify what the behavior of omitting the keyword is.  (Either that or remove all other omission clauses; probably not.)

Reply:
Compare to [`allOf`](https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-01#name-allof) and friends, just removing it is consistent with how they are. Plenty of other keywords don't include a comparable sentence.